src/core/DirectGeometry.js

```diff
@@ -103,7 +103,7 @@ Object.assign( DirectGeometry.prototype, {
 
 			for ( var i = 0; i < morphTargetsLength; i ++ ) {
 
-				morphTargetsPosition[ i ] = [];
+				morphTargetsPosition[ i ] = { name: morphTargets[i].name, array: [] };
```

Comment:
Please add spaces here `morphTargets[i].name`. It should be `morphTargets[ i ].name`

Reply:
spaces added.